clusterloader2/pkg/measurement/common/dns/dns_performance-k8s-hostnames.go

```diff
@@ -182,6 +221,35 @@ func (m *dnsPerfK8sHostnamesMeasurement) deleteDNSClientPermissions() error {
 	return m.k8sClient.RbacV1().ClusterRoleBindings().Delete(context.TODO(), dnsPerfTestPermissionsName, metav1.DeleteOptions{})
 }
 
+func (m *dnsPerfK8sHostnamesMeasurement) deleteDNSClient() error {
```

Comment:
The clients are deleted when the namespace is deleted in the cleanup phase.

I assume it's best practice to separately clean up the pods to identify any issues with the clean up.

Reply:
Added cleanup code to remove the additional manifests